Examples/Apply_scipy_function_to_every_gridpoint.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    years_since_start = xr.DataArray(np.arange(sst.time.shape[0])/12,
</pre><p> </p><p>this reads hardcoded... what does</p><p> </p>

<pre class="ql-syntax" spellcheck="false">np.arange(sst.time.shape[0])/12
</pre><p> </p><p>mean to do? will it work in general? only with monthly data?</p><p> </p><p>Could we do this more generally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>